BuildProject/tests/App.test.js

```diff
@@ -0,0 +1,37 @@
+import wd from 'wd';
```

Comment:
Do you want the file to a `.ts`?

Reply:
I looked into this for about 30 minutes and can't get it to work. I don't think I want to spend more time on this, since it was working before. Sorry. :-(